src/Options/index.js

```diff
@@ -124,6 +124,9 @@ export interface ParseServerOptions {
   preventLoginWithUnverifiedEmail: ?boolean;
   /* Email verification token validity duration, in seconds */
   emailVerifyTokenValidityDuration: ?number;
+  /* an existing password reset token should be reused when resend verification is requested
+  :DEFAULT: false */
+  emailVerifyTokenReuseIfValid: ?boolean;
   /* account lockout policy for failed login attempts */
```

Comment:
It seems that the `resetTokenReuseIfValid ` option is missing; can you please add it to the `passwordPolicy` key group?

Reply:
As far as I can tell, there aren't any definitions for passwordPolicy. I was going to tackle that in #7015.

Reply:
OK, well spotted. However, we should also add validation for new config parameters, even if they are lacking explicit definition. Can you please add the validation for `resetTokenReuseIfValid` here:

https://github.com/parse-community/parse-server/blob/e6ac3b6932186a6a85601c65143305f86f63e948/src/Config.js#L146

And add the validation for `emailVerifyTokenReuseIfValid ` here:
https://github.com/parse-community/parse-server/blob/e6ac3b6932186a6a85601c65143305f86f63e948/src/Config.js#L205

Reply:
No worries, thanks for the pointers again 😊